ds_judgements_public_ui/sass/includes/_cards_homepage.scss

```diff
@@ -2,49 +2,81 @@
   @include container;
   list-style-type: none;
   margin: $space-6 auto;
-  @media (min-width: $grid-breakpoint-small) {
-    margin: $space-12 auto;
-  }
   padding: 0;
   display: flex;
   flex-direction: column;
-  gap: $space-12;
-  @media (min-width: $grid-breakpoint-medium) {
-    flex-direction: row;
-    li {
-      width: 33%;
+  gap: 0;
```

Comment:
I think these cards need a bit of space between them.

This is the design:
<img width="511" alt="image" src="https://github.com/user-attachments/assets/8965693c-1a72-4178-a081-d61485fb6b9a">

This is how it currently looks:
<img width="477" alt="image" src="https://github.com/user-attachments/assets/d33538f9-dea7-438d-a34b-9315a9ae88da">

Reply:
@jlhdxw updated text and the image Terry added as JIra card attachment  is now not green arrows as the design